content/en/synthetics/browser_tests.md

```diff
@@ -140,25 +140,41 @@ To use your variables in one of your assertions, hit *Use Variable* and select t
 
 {{< img src="synthetics/browser_tests/use_variable_in_assertion.png" alt="Use variable in assertion" responsive="true" style="width:40%;">}}
 
-#### Wait
+### Wait
 
-By default, Datadog waits for a page to be fully loaded before performing an action or a next step—with a timeout after 60 seconds. In some cases, however, you may wish to set a custom waiting time. For instance, if you know that a page or a page element is taking more than 60 seconds to load, you can leverage the wait step in order to extend that default timeout. If you choose to use this functionality, the value for your wait step must not exceed 300 seconds. 
+By default, Datadog waits for a page to be fully loaded before performing an action or a next step—with a timeout after 60 seconds. In some cases, however, you may wish to set a custom waiting time. For instance, if you know that a page or a page element is taking more than 60 seconds to load, you can leverage the wait step in order to extend that default timeout. If you choose to use this functionality, the value for your wait step must not exceed 300 seconds.
 
 **Note**: This additional time is systematically added to **each run** of your browser test scenario.
 
-#### Test failure and errors
+### Subtests
+
+You can run browser tests within other browser tests:
+
+{{< img src="synthetics/browser_tests/browser_test_subtest.mp4" alt="Browser test subtest" video="true" responsive="true" width="40%" >}}
+
+Advanced options (optional) allow you to choose where you want your subtest to be played:
+
+* **Main**: Subtest is played in your main tab, in sequence with other steps. This is the default option.
+* **New**: Subtest is played in a new tab, which will be closed at the end of the subtest (and thus cannot be re-used).
+* **Specific tab**: Subtest is played in a numbered tab which can be reused in other steps outside of the subtest.
+
+Opening your subtest in the main tab means that your subtest is the continuation of your main test: it uses the URL from the previous step (i.e. the start URL of your main test if subtest is set as a first step and the last page URL otherwise). Opening your subtest in a new tab or in a specific tab means that the test starts running from the subtest start URL.
```

Comment:
Maybe remove the `(i.e. the start URL of your main test if subtest is set as a first step and the last page URL otherwise)`, I think it was already well explained before and this just complicates a bit?